vm_supervisor/run.py

```diff
@@ -216,3 +214,30 @@ async def run_code_on_event(vm_hash: VmHash, event, pubsub: PubSub):
             execution.stop_after_timeout(timeout=settings.REUSE_TIMEOUT)
         else:
             await execution.stop()
+
```

Comment:
Unrelated to the PR but I opened the project in PyCharm and I had a nice warning about execution.stop_after_timeout not being awaited, so that's probably a bug you want to take care of.

Reply:
I had the same, it is a PyCharm error, probably due to the return value of the following definition:
```python
def stop_after_timeout(self, timeout: float = 5.0) -> Optional[Task]:
```